src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts

```diff
@@ -199,6 +203,24 @@ export class InlineCompletionsController extends Disposable {
 		this._register(new InlineCompletionsHintsWidget(this.editor, this.model, this.instantiationService));
 	}
 
+	private provideScreenReaderHint(): void {
+		const showHoverKeybinding = this._keybindingService.lookupKeybinding('editor.action.showHover');
+		const accessibleViewKeybinding = this._keybindingService.lookupKeybinding('editor.action.accessibleView');
+		if (this.configurationService.getValue('accessibility.verbosity.inlineCompletions')) {
+			let hint: string | undefined;
+			if (showHoverKeybinding && accessibleViewKeybinding) {
+				hint = localize('showBothHints', "View more actions ({0}) or inspect this in the accessible view ({1})", showHoverKeybinding.getAriaLabel(), accessibleViewKeybinding.getAriaLabel());
```

Comment:
TODO: do we even need to inform them about the hover? I don't think so as it basically does what the accessible view will do, but doesn't include the text

Reply:
I think I agree, we don't need to advertise the hover if the accessible view offers the same feature set.